ui/components/ui/icon-with-fallback/icon-with-fallback.component.js

```diff
@@ -23,14 +23,14 @@ const IconWithFallback = ({
       src={icon}
       style={style}
       className={className}
-      alt={name.length ? name : 'icon'}
+      alt={name || 'icon'}
       {...props}
     />
   ) : (
     <span
       className={classnames('icon-with-fallback__fallback', fallbackClassName)}
     >
-      {name.length ? name.charAt(0).toUpperCase() : ''}
+      {name && name.length ? name.charAt(0).toUpperCase() : ''}
```

Comment:
Could this be `name?.length` instead?

Reply:
Initially I thought not but that would be equivalent. I think we should merge this anyway because really our linter should handle this sort of thing :thinking: